lib/spack/spack/test/database.py

```diff
@@ -111,9 +111,6 @@ def test_default_queries(database):
     libraries = spec['zmpi'].libs
     assert len(libraries) == 1
 
-    cppflags_expected = '-I' + spec.prefix.include
-    assert spec['zmpi'].cppflags == cppflags_expected
```

Comment:
@adamjstewart: You could change what `install --fake` puts in the target directory to include a dummy header (it contains a dummy library at the moment -- not sure about the header).